### ISSUE TITLE
Delete invalid confidence in ModelExecutor

### ISSUE THREAD

**Issue by zhaochenyang20** [neulab/prompt2model#173]
Actually, confidence for conditional generation is not a well-defined problem. As far as I know, only in classification tasks, there is a clear definition of confidence.

And even for classification, the problem is that since all the tasks are converted to generation in our pipeline, makes the confidence hard to compute. Actually, even for the classification task, our pipeline does not specify the class range. It’s hard for us to know how many classes are here.

Additionally, our current way of computing confidence is actually invalid.

```
logits = output[idx].float()
probs = torch.softmax(logits, dim=-1)
confidence = probs.mean().item()
```

Here we are computing the mean of softmax probs. The mean should always be 1/n. My idea is to delete the confidence.
